### DISCUSSION TITLE
Building docker on macOS seems to have dependency issues

### DISCUSSION THREAD

**Discussion by EdanToledo** [instadeepai/Mava#829] [Q&A]
Hello guys, 

Trying to build docker image on macOS and it seemingly is giving me a reverb versioning issue. Not sure if this is just me or there's been some updates to reverb which has caused some breaking issues to the docker build.

This is the error I get:
```
#13 5.365 ERROR: Ignored the following versions that require a different python version: 0.7 Requires-Python >=3.6, <3.7; 0.8 Requires-Python >=3.6, <3.7
#13 5.365 ERROR: Could not find a version that satisfies the requirement dm-reverb~=0.7.2; extra == "reverb" (from id-mava[reverb]) (from versions: none)
#13 5.365 ERROR: No matching distribution found for dm-reverb~=0.7.2; extra == "reverb"
```

I get this when running `make build version=jax-core`

Even when removing the specific dm-reverb version in the setup.py file the issue persists.

**Comment by DriesSmit** [comment]
Hey @EdanToledo :slightly_smiling_face: 

@KaleabTessera you are using a macOS right?

**Reply by EdanToledo** [reply]
There seems to be problems when using an M1 - if emulating x86 architecture using docker then I believe there is a special way to install jax that currently is not being done. If using an arm64 version of ubuntu - you can't install reverb and launchpad. Not sure what the solution is? probably emulation with weird jax installation.

**Comment by sash-a** [comment]
Try reverb~=0.7.3, anything in 0.7 should work with minor tweaks, maybe it just can't find that exact version for some reason?

Otherwise I think you might need to do the weird jax install for M1 :cry:
